qiskit/algorithms/evolvers/variational/solvers/__init__.py

```diff
@@ -0,0 +1,44 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021, 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""
+Solvers (:mod:`qiskit.algorithms.evolvers.variational.solvers`)
```

Comment:
This is the docstring in solvers that I mentioned that I do not see referenced for inclusion by Sphinx into the overall API ref.

Reply:
It is used internally. In such a case, do you suggest removing this docstring or can we keep it as a documentation for developers?

Reply:
Lets leave it there for now. I am not sure we really have a distinction in the API ref for end users that use the code as-is, versus developers that may extend it. Some classes get seen by both communities, one may use the methods it defines, another may implement them in some derived class. If it warrants linking into the main API ref later at least the text will be there ready, and in the interim at least it will be there in the source.